### PR TITLE
QT zephyr build fixes

### PR DESCRIPTION
**Pull request by D-Triveni**

### CODE REVIEW

File: samples/wifi/wfa_qt_app/src/qt_app_main.c [hunk 1]
```diff
@@ -33,6 +75,29 @@ int main(void)
...
+		net_if_ipv4_addr_add(wifi_iface, &addr, NET_ADDR_MANUAL, 0);
+	}
+	if (sizeof(CONFIG_NET_CONFIG_MY_IPV4_NETMASK) > 1) {
+		/* If not empty */
+		if (net_addr_pton(AF_INET, CONFIG_NET_CONFIG_MY_IPV4_NETMASK, &addr)) {
+			printk("Invalid netmask: %s", CONFIG_NET_CONFIG_MY_IPV4_NETMASK);
+		} else {
+			net_if_ipv4_set_netmask(wifi_iface, &addr);
+		}
+	}
```
Comment (krish2718) [inline samples/wifi/wfa_qt_app/src/qt_app_main.c:98]: Same IP for both USB and Wi-Fi?

**Review by krish2718 — CHANGES_REQUESTED** [review]

File: samples/wifi/wfa_qt_app/src/qt_app_main.c (hunk 1 above)
Comment (D-Triveni) [reply samples/wifi/wfa_qt_app/src/qt_app_main.c:98]: No

File: samples/wifi/wfa_qt_app/prj.conf [hunk 2]
```diff
@@ -6,6 +6,7 @@
 CONFIG_WIFI=y
 CONFIG_WIFI_NRF700X=y
 CONFIG_WFA_QT_CONTROL_APP=y
+CONFIG_PTHREAD_IPC=y
```
[the remark this answers is not in the document]
Comment (D-Triveni) [reply samples/wifi/wfa_qt_app/prj.conf]: In QT, I added to select the flag. not added as dependent.

**Review by krish2718 — APPROVED** [review]

File: samples/wifi/wfa_qt_app/prj.conf (hunk 2 above)
[the remark this answers is not in the document]
Comment (krish2718) [reply samples/wifi/wfa_qt_app/prj.conf]: `select` will automatically choose the flag.

File: samples/wifi/wfa_qt_app/src/qt_app_main.c [hunk 3]
```diff
@@ -16,15 +16,64 @@
...
+	init_usb();
+	/* Redirect static IP address to netusb*/
+	const struct device *usb_dev = device_get_binding("eth_netusb");
+	struct net_if *iface = net_if_lookup_by_dev(usb_dev);
+
+	if (!iface) {
+		printk("Cannot find network interface: %s", "eth_netusb");
+		return -1;
+	}
+	if (sizeof(CONFIG_NET_CONFIG_USB_IPV4_ADDR) > 1) {
```
Comment (krish2718) [inline samples/wifi/wfa_qt_app/src/qt_app_main.c:60]: You can use the default static IP address for USB and DHCP for Wi-Fi.

Comment (D-Triveni) [reply samples/wifi/wfa_qt_app/src/qt_app_main.c:55]: In QT, DHCP configuration is not used. So, keeping static configuration only.

**Review by krish2718 — APPROVED** [review]

**Review by bama-nordic — APPROVED** [review]

**Pull request merged**